automation/jenkins_build.sh

```diff
@@ -218,22 +218,29 @@ if [ "$deploy" == "yes" ]; then
 		-e SLUG="$SLUG" \
 		-e BUILD_VERSION="$S3_VERSION_HOSTOS" \
 		-e DEVELOPMENT_IMAGE="$DEVELOPMENT_IMAGE" \
+		-e DEPLOYER_UID=$(id -u) \
```

Comment:
`The image developer can create additional users. Those users are accessible by name.` This sounds like you actually need to create them first in the image. But then you need to install packages which you need to do with sudo so if we gain the ability of not needing to su we will need to include sudo and use it for installing the other packages. Actually this won't work at all. Because sudo is not installed in the image and installing it needs root.